scripts/makefile/composer-validation.sh

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env sh
+set -e
 
 # Get command result and its exit code
-COMMAND=$(composer validate --strict --quiet)
+COMMAND=$(docker run --rm -v `pwd`:`pwd` -w `pwd` skilldlabs/php:72 composer validate --strict --quiet)
```

Comment:
it is very weird that docker image is hardcoded here, that's why I was against using bash scripts instead of Makefiles

Reply:
Indeed, now fixed!